flink-python/pyflink/fn_execution/coders.py

```diff
@@ -195,6 +287,22 @@ def get_impl(self):
     def __repr__(self):
         return 'RowCoder[%s]' % ', '.join(str(c) for c in self._field_coders)
 
+    @staticmethod
+    def from_schema_proto(schema_proto):
```

Comment:
I don't find where this method is called, so I think the `RowCoder` doesn't need to extend `BaseCoder`